_posts/2017-12-01-ome-files-0-5-0.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Release of OME Files C++ 0.5.0
+intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.5.0
+---
+OME Files is a reference implementation of the OME Data Model and
+OME-TIFF file format for the storage and interchange of biological
+imaging data and metadata.  It provides support for:
+
+- the OME Data Model (model objects, metadata store, XSL transforms
+  and XML validation)
+- OME-TIFF reading
+- OME-TIFF writing
+
+The main change in this release is the re-addition of compatibility
+code to permit building on CentOS6 and RHEL6 with EPEL.  Other work
```

Comment:
Do we want to mention you can build on CentOS 7 with the package dependencies? CentOS 6 still requires the superbuild right?

Reply:
I'm not 100% sure.  We haven't tested CentOS 6 with EPEL, but it might work.  I can't remember if there were any blockers on CentOS 6.  I can certainly reword it to remove CentOS 6 here until we have confirmed this.

Reply:
:+1: for putting the emphasis on CentOS 7 as we have definitely tested it.